curriculum/challenges/english/25-front-end-development/review-css/671a9a0a140c2b9d6a75629f.md

```diff
@@ -19,7 +19,7 @@ Review the concepts below to prepare for the upcoming exam.
 ## Inline, Internal, and External CSS
 
 - **Inline CSS**: These styles are written directly within an HTML element using the `style` attribute. Most of the time you will not be using inline CSS due to separation of concerns. 
-- **Internal CSS**: These styles are written within the `<style>` tags inside the `<head>` section of an HTML document. This can be useful for creating short code examples, but usually you will not need be using internal CSS.
+- **Internal CSS**: These styles are written within the `<style>` tags inside the `<head>` section of an HTML document. This can be useful for creating short code examples, but usually, you will not to use internal CSS.
```

Comment:
```suggestion
- **Internal CSS**: These styles are written within the `<style>` tags inside the `<head>` section of an HTML document. This can be useful for creating short code examples, but usually, you will not use internal CSS.
```